optuna/study.py

```diff
@@ -11,6 +11,7 @@
 
 from optuna._experimental import experimental
 from optuna._study_direction import StudyDirection
+from optuna._study_summary import StudySummary
```

Comment:
Oh, maybe we need `# NOQA` here as the symbol is not used within this file.
```suggestion
from optuna._study_summary import StudySummary  # NOQA
```

Reply:
Oh! Yes, you are right. In addition, it is only used in type hinting, so I will move the import statement below the TYPE_CHECKING block.